apache/superset
# feat: Support further drill by in the modal

### SUMMARY
Before, user could only drill by once, from the original chart. This PR implements  further drilling in the drill by modal. In drill by modal, only drill by action is available - drill to detail and cross filtering is not available.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://user-images.githubusercontent.com/15073128/230602328-d32c18da-b432-4a6f-a1c5-7ab1553fdc03.mov



### TESTING INSTRUCTIONS
1. Enable `DRILL_BY` ff
2. Add a supported chart with groupby to a dashboard
3. Drill by a column
4. In the drill by modal, right click on a series to further drill by
5. The result of further group by should render in the same modal
6. The columns available for drilling should NOT contain columns that already were used for drilling before

### ADDITIONAL INFORMATION
